packages/driver/cypress/fixtures/nested/with_paths.css

```diff
@@ -1,4 +1,4 @@
 @font-face {
   font-family: 'Some Font';
-  src: url('../fonts/some-font.eot?#iefix') format('embedded-opentype'), url('../fonts/some-font.woff2') format('woff2'), url('../fonts/some-font.woff') format('woff'), url('../fonts/some-font.ttf') format('truetype'), url('../fonts/some-font.svg#glyphicons_halflingsregular') format('svg');
```

Comment:
If we explicitly set these fonts, and hey aren't coming through in our tests, any concerns?

Reply:
I think it's ok because chrome wouldn't honor them anyway?

Reply:
SGTM.